projects/approved-premises-and-delius/src/main/kotlin/uk/gov/justice/digital/hmpps/model/CaseDetail.kt

```diff
@@ -38,7 +38,14 @@ data class Profile(
 
 data class Manager(val team: Team)
 
-data class Team(val code: String, val name: String, val ldu: Ldu)
+data class Team(
+    val code: String, val name: String, val ldu: Ldu,
+    val borough: Borough?,
```

Comment:
Doesn't matter too much, but I wonder if we should avoid the legacy terminology such as probationArea / borough / ldu in the response field names.  These are now called provider / probation delivery unit / local admin unit in Delius - https://ministryofjustice.github.io/hmpps-probation-integration-services/tech-docs/concepts.html#provider